pkg/admission/validator/shoot.go

```diff
@@ -59,6 +59,10 @@ func (s *shootValidator) validateShoot(_ context.Context, shoot *core.Shoot) err
 		return field.TooMany(fldPath.Child("rule", "cidrs"), len(extensionSpec.Rule.Cidrs), DefaultAddOptions.MaxAllowedCIDRs)
```

Comment:
Can we move the `MaxAllowedCIDRs` validation to `ValidateExtensionSpec`?

Reply:
We could consider hard-coding it to a reasonable value. I don't know why we should make it configurable 😄

Reply:
Ah that may be too disruptive. I've changed the code so that both variants stay in place and get checked properly.